async-indexing/src/main/java/zone/cogni/asquare/service/index/GraphIndexService.java

```diff
@@ -109,6 +104,24 @@ else if(StringUtils.isNotBlank(resourceIndex.getGraph())) {
     return false;
   }
 
+  private void executeFacetConversion(ResourceIndex resourceIndex, TypedResource resource, ObjectNode json) {
+    if (facetConversionSupplier != null) {
```

Comment:
I do not like this. See for example the book "Refactoring" -> https://silab.fon.bg.ac.rs/wp-content/uploads/2016/10/Refactoring-Improving-the-Design-of-Existing-Code-Addison-Wesley-Professional-1999.pdf

Lookup chapter: Replace Nested Conditional with Guard Clauses
A method has conditional behavior that does not make clear the normal path of execution.

Reply:
i changed it to make it more readable